tools/remote_build/windows.bazelrc

```diff
@@ -38,7 +37,7 @@ test --test_env=GRPC_VERBOSITY=debug
 
 # Set flags for uploading to BES in order to view results in the Bazel Build
 # Results UI.
-build --bes_backend="buildeventservice.googleapis.com"
+build --bes_backend=grpcs://buildeventservice.googleapis.com
```

Comment:
![image](https://user-images.githubusercontent.com/7281574/64634223-9bea3380-d3b1-11e9-89d4-d80bd2e2b1df.png)

Reply:
I love this reaction image, but I have no idea how to interpret it. :joy: